docs/sources/guides/whats-new-in-v7-2.md

```diff
@@ -28,19 +27,16 @@ The main highlights are:
   - [**Organization-wide report settings**]({{< relref "#organization-wide-report-settings" >}})
   - [**Report grid layout**]({{< relref "#report-grid-layout" >}})
 
-## Date formatting options
+## New date formatting options added
```

Comment:
if you update a heading you need to update the link to this section as well (at the top of the article we list an overview with links to each section)

Reply:
Good catch, thank you!